shell/edit/provisioning.cattle.io.cluster/tabs/Advanced.vue

```diff
@@ -101,14 +115,48 @@ export default {
           </template>
           <h3 v-else>
             {{ advancedTitleAlt }}
+            <i
+              v-if="kubeletArgTooltip"
+              v-clean-tooltip="kubeletArgTooltip"
+              class="icon icon-info"
+            />
           </h3>
 
           <ArrayList
+            v-if="i === 0 && serverConfig['kubelet-arg']"
+            v-model="serverConfig['kubelet-arg']"
+            class="mb-10"
+            data-testid="global-kubelet-arg"
+            :mode="mode"
+            :add-label="t('cluster.advanced.argInfo.machineGlobal.listLabel')"
+          >
+            <template #empty>
+              {{ '' }}
+            </template>
+          </ArrayList>
+
+          <ArrayList
+            v-if="row.value.config && (row.value.config['kubelet-arg'] || !serverConfig['kubelet-arg'])"
```

Comment:
i need to play around with this more, but i think it should be below  i.e. show if there IS a machineSelectorConfig without a selector and with 'kubelet-arg' or there is no machineGlobalConfig a 'kubelet-arg'

```
v-if="row.value.config && ((agentConfig || {})['kubelet-arg'] || !serverConfig['kubelet-arg'])"
```

to test i've been tweaking 

```
  created() {
    this.value.spec.rkeConfig.machineGlobalConfig['kubelet-arg'] = ['imfromglobal'];
    this.value.spec.rkeConfig.machineSelectorConfig[0]['kubelet-arg'] = ['imfromselector'];
  },
```

Reply:
I changed `row.value.config['kubelet-arg']` -> `(agentConfig || {})['kubelet-arg']`

Reply:
That change has been reverted, how come?

Reply:
@richard-cox 

- I had a quick look to the `agentConfig` value and it's the same of row.value.config in this case; it's tested in the unit test though
-  The unit tests fail when using `agentConfig`, since it's a computed value which triggers a DOM update not cacthed in time by unit test.